src/components/link.js

```diff
@@ -49,6 +49,11 @@ export default {
         // don't redirect on right click
         /* istanbul ignore if */
         if (e.button !== 0) return
+        // don't redirect if `target="_blank"`
+        /* istanbul ignore if */
+        const target = this.$el && this.$el.getAttribute('target')
+        if (target && target.toLowerCase().split(' ').indexOf('_blank') > -1) return
```

Comment:
Since code is written in ES Net, shouldn't it be `target.toLowerCase().split(' ').includes('_blank')`?